examples/sqlserverflex/sqlserverflex.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/stackitcloud/stackit-sdk-go/core/config"
+	"github.com/stackitcloud/stackit-sdk-go/core/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/sqlserverflex"
+	"github.com/stackitcloud/stackit-sdk-go/services/sqlserverflex/wait"
+)
+
+func main() {
+	// Specify the project ID
+	projectId := "16f49d71-37ad-4137-8b97-44d9c55c4094"
```

Comment:
This should be a placeholder

Reply:
Ups, these were leftovers from testing. Didn't mean to commit them